### ISSUE TITLE
Bug Report Page - Clear and easy to access bug reporting instructions

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea-game.github.io#17]
There is enough information and detail around bug reports that it could use it's own page. Maybe under website top level "help" category. IE:   http://triplea.../helps/ReportingBugs.html

Page would contain the details of what to include in the report, and then some links and maybe screenshots of actually how to submit the report.

I'm curious if others agree there is enough detail around bug reports that it could use it's own treatment. I could even see reason to have a "report a bug" link in the header links.

**Comment by williamdekryger** [comment]
How do we now want users to report bugs? Github issues?

**Comment by DanVanAtta** [comment]
Good question. My understanding is that sourceforge ticket queue is being viewed as legacy. Most of the instructions will be about what to include a bug report. For now IMO let's point people to github issues. Worst case we update it to refer folks to source forge.

**Comment by DanVanAtta** [comment]
We need to make it "super easy" to know how to submit bug reports. It's a huge process gap for us at the moment. Labelling this as urgent to reflect the higher priority.

**Comment by williamdekryger** [comment]
@DanVanAtta I have written a page for this. It can be found in the bug_reporting_page branch. However, I would like a dev to make sure I covered all of my bases.

First, I linked to it from the contribute page:
![link-to-bug-report-instructions](https://cloud.githubusercontent.com/assets/5250014/13230766/944ac52e-d99e-11e5-8148-144197078567.png)

And here is the page itself:
![bug-reporting-instructions](https://cloud.githubusercontent.com/assets/5250014/13230783/a25a59fe-d99e-11e5-911d-e48f2cef142e.png)

**Comment by DanVanAtta** [comment]
Overall looks good. Some more instructions may be needed on how to find the java error console, and the AI logs, and how/where to post files. Zenhub is not by default enabled for just anyone, so not everyone will be able to initially attach files to issues.

**Comment by DanVanAtta** [comment]
@williamdekryger , screenshots look good. But I'm noticing the link is not where I would expect it. I was expecting it to be under 'help & documentation'.

Please make the link from that page if you would, and have it be very prominent / easy to find. IMO the downloading game flow and reporting problems flow are the two most important. Next release we'll want to herd the user base to download the updated game through the website.

**Comment by williamdekryger** [comment]
@DanVanAtta Sure, I'll be taking care of that.

**Comment by DanVanAtta** [comment]
@williamdekryger , is this still in progress?

**Comment by williamdekryger** [comment]
@DanVanAtta I still have the work in progress for this on my local repo in a separate branch. I'll need to work on getting that merged with the latest changes.

**Comment by DanVanAtta** [comment]
Closing, missing action items in this ticket, it's a bit stalled. #250 has an update to the bug report page. 

One intent behind this tracking ticket is that we take a full overview of the bug reporting process. I think at the moment that is no longer necessary. We have consolidated the help links from the game to the website, and we just slowly updated that. I think the real next steps will be to start building bug reporting and tracking into the game, so no external website would be needed, and crash info is automatically sent to us.
